### DISCUSSION TITLE
¡Ayuda! comando git config -h

### DISCUSSION THREAD

**Discussion by glucambio** [community/community#123977] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

![Captura de pantalla 2024-05-15 a la(s) 2 01 26 p  m](https://github.com/community/community/assets/101686035/5f4ea777-d9f7-400b-b8a7-31516103ee68)
Hola amigos quien pueda ayudarme necesito ayuda con esto que me pasa en git. Al momento de ingresar el comando git config -h me genera el error que aparece en la imagen ese error no se por qué sera y en la termina de mi mac y en la terminar warp me sale el mismo error. Podrían ayudarme con esto.
![Captura de pantalla 2024-05-15 a la(s) 2 01 26 p  m](https://github.com/community/community/assets/101686035/70f03a90-3d69-46c5-99e2-c67ae0907002)

**Comment by davevad93** [comment, marked as answer]
Hi @glucambio ,

That is not an error, when you run `git config -h` it displays a summary for the `git config` command with its options and usage examples. It's a  guide that explains how to use the `git config` command based on what you need to do.

![4](https://github.com/community/community/assets/94308234/0522913f-5584-4a7c-a662-c70058ec4511)

**Reply by glucambio** [reply]
Muchas gracias amigo. Pensé que era un error ya que veía un curso por youtube de git y a la persona no le salía --[no- ] y otra ayuda amigo sabes como como configuro el git a ingles por que esta en español y no se si me creara problemas a futuro.
![Uploading Captura de pantalla 2024-05-15 a la(s) 3.34.43 p. m..png…]()

**Reply by glucambio** [reply]
![Uploading Captura de pantalla 2024-05-15 a la(s) 3.34.43 p. m..png…]()

**Reply by davevad93** [reply]
You're welcome @glucambio , I don't think it matters the language since it is only for displaying messages in the terminal, anyway if you want to change it to English I'll leave you a link that explains how to do it in Linux systems, take a look at the answer of this post:

- [How does one change the language of the command line interface of Git?](https://stackoverflow.com/questions/10633564/how-does-one-change-the-language-of-the-command-line-interface-of-git)

**Reply by davevad93** [reply]
P.S. I could easily answer you in Spanish since I live in Spain, the reason why I answer you in English is because right now only English is allowed in discussions, the reason is that GitHub currently has no moderators for languages other than English. Keep that in mind for the future because the Community moderators are locking any discussion in non-English languages.

**Reply by glucambio** [reply]
Muchas gracias amigo me fuiste de mucha ayuda de verdad estaba estancado en eso. Okey no sabia eso lo de hablar en ingles por acá, pero lo tendré en cuenta para la próxima.

**Reply by davevad93** [reply]
You're welcome, If you think I've answered properly to your question, please mark my first answer as `answered` so it can help other people in the future who may have the same doubt, buenas noches @glucambio !

**Comment by ebndev** [comment]
HI @glucambio! I'm glad you were able to get an answer to your post.

As @davevad93 mentioned, unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create new discussions in English if you have questions in the future!
